frontend/src/concepts/k8s/utils.ts

```diff
@@ -44,12 +46,13 @@ type AdditionalCriteriaApplied = Record<
  */
 export const translateDisplayNameForK8sAndReport = (
   name: string,
-  { safeK8sPrefix, maxLength }: AdditionalCriteriaForTranslation = {},
+  { safeK8sPrefix, isStaticPrefix, maxLength }: AdditionalCriteriaForTranslation = {},
 ): [string, AdditionalCriteriaApplied] => {
   const appliedCriteria: AdditionalCriteriaApplied = {
     autoGenerated: false,
     safeK8sPrefix: false,
     maxLength: false,
+    isStaticPrefix: false,
```

Comment:
```suggestion
    staticPrefix: false,
```
Maybe the wording here is weird -- they are all booleans, because they are status of what happened... `appliedCriteria.staticPrefix` imo is better than `appliedCriteria.isStaticPrefix` -- it's all the applied criteria so everything is prefixed with `is` or none of them are.

Reply:
@christianvogt this is not a blocker to me... but are you in the middle of doing this?

Reply:
I'll fix this. I agree with you.

Reply:
Added a new commit with just the rename of this.